.scaffold/tests/bats/deployment--artifact.bats

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bats
+#
+# Test for CircleCI lifecycle.
+#
+# shellcheck disable=SC2030,SC2031,SC2129,SC2155
+
+load _helper.bash
+load _helper.deployment.bash
+
+
+@test "Missing or Invalid DREVOPS_DEPLOY_TYPES" {
+  substep "Swap to ${LOCAL_REPO_DIR}"
+  pushd "${LOCAL_REPO_DIR}" >/dev/null || exit 1
+
+  export DREVOPS_DEPLOY_TYPES=""
+  run ahoy deploy
+  assert_failure
+
+  assert_output_contains "Missing required value for DREVOPS_DEPLOY_TYPES. Must be a combination of comma-separated values (to support multiple deployments): code, docker, webhook, lagoon."
+
+  popd >/dev/null
+}
+
+@test "SSH Key Setup script" {
```

Comment:
This would be 
```
"Valid SSH Key, MD5 fingerprint" {
```

Then, there would be other cases:
- `"Valid SSH Key, SHA256 fingerprint"` {`
- `"Invalid SSH Key, MD5 fingerprint"` {`
- `"Invalid SSH Key, SHA256 fingerprint"` {`
- `"Valid SSH Key, SHA256 fingerprint, file from env var"` {` - in this case, any discovered keys with provided fingerprints would not be use and the key file name from the env variable will be used instead.

Reply:
ok nice i will adapt it to this method